songs/package.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "adiherzog.github.io",
+    "version": "0.0.2",
+    "description": "",
+    "main": "index.html",
+    "scripts": {
+        "test": "eslint './**/*.js'"
+    },
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/xeronimus/adiherzog.github.io.git"
+    },
+    "keywords": [],
+    "author": "",
+    "license": "ISC",
```

Comment:
It's ok for me to use ISC as a license. Please also add a LICENCE file into the "songs" folder.